api/openapi-spec/swagger.json

```diff
@@ -3080,8 +3080,7 @@
     "description": "CDI is the CDI Operator CRD",
     "type": "object",
     "required": [
-     "spec",
-     "status"
+     "spec"
```

Comment:
Just a question: where from do we get api/openapi-spec/swagger.json, how it compares to 
- pkg/apis/core/v1alpha1/openapi_generated.go and 
- pkg/apis/core/v1beta1/types_swagger_generated.go.

Reply:
Its generated during make generate by openapi-spec-generator that exists in <cdi repo>/bin

Reply:
And it is compiled by the cdi builder and put there during the rsync that happens when we compile things in the build container.